UnityProject/Assets/Scripts/UI/Objects/Command/IDConsole/GUI_IDConsole.cs

```diff
@@ -213,7 +213,7 @@ public void ServerRemoveTargetCard(ConnectedPlayer player)
 
 		public void ServerRemoveAccessCard(ConnectedPlayer player)
 		{
-			if (console.AccessCard == null || IsAIInteracting() == false)
+			if (console.AccessCard == null)
 			{
```

Comment:
What will this do for the si?

Reply:
AI*

Reply:
yeah, but it doesn't make any sense since it would trigger with normal players to , and I can't find a way of to check if it's a AI or not